openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#
+#  sb-signing-utils
+#
+################################################################################
+
+SB_SIGNING_UTILS_SITE ?= $(call github,hellerda,sb-signing-utils,$(SB_SIGNING_UTILS_VERSION))
+SB_SIGNING_UTILS_SITE_VERSION = master
```

Comment:
Removed the unnecessary line.  

Agreed, will tag it when it gets merged into open-power/sb-signing-utils.  For the moment it's pointing to a specific commit id (via SB_SIGNING_UTILS_VERSION, not shown here) in my private sb-signing-utils branch.